_docs/cardano/2017-02-07-topology.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Topology
+permalink: /cardano/topology/
+group: cardano
+---
+<!-- Reviewed at dec0d911d6c4beb8e708ed4076f832ff871f6125 -->
+
+# Cardano SL Network Topology
+
+This is an overview of Cardano SL network topology.
+
+## Nodes Groups
+
+Cardano SL is a distributed network: it consists of nodes. But currently all these nodes are divided into
+3 groups:
+
+1. Core
+2. Relay
+3. Edge
+
+Let's describe each of these groups.
+
+### Core Node
+
+You can think of core nodes as of Cardano SL core.
+
+These nodes are the most important ones: if they are alive, the blockchain cannot be compromised. So core
+nodes are isolated: they are fully under the control of the IOHK and cannot be directly accessible from
+the public internet. The reason of such an isolation is a DoS-protection: if nodes cannot be directly
+accessible - they cannot be directly attacked.
```

Comment:
> they cannot be directly attacked

remove this and replace with something like we reduce probability from them being attacked